UPGRADING

```diff
@@ -92,6 +92,11 @@ PHP 8.5 UPGRADE NOTES
 - DOM:
   . Added Dom\Element::$outerHTML.
 
+- Standard:
+  . getimagesize() now supports SVG images when ext-libxml is also loaded.
+    Similarly, image_type_to_extension() and image_type_to_extension()
+    now also handle IMAGETYPE_SVG.
```

Comment:
Maybe add a note about the new array element `width_unit` and `height_unit` (for the doc people).

Reply:
Of course